### PR TITLE
Feature/annofuse:v0.91.0

### PR DESCRIPTION
**Pull request by sakshamphul**
## Description

This PR brings in a docker for latest version of annofuse.

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

This was test locally by building the docker image and running annofuse within the container. It takes close to 15 mins to build this images

## Checklist:

- [X] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings
- [X] I have committed any related changes to the PR

### CODE REVIEW

**Review by luederm — APPROVED** [review]
Looks good although I don't know enough to review the circleci config

**Review by migbro — CHANGES_REQUESTED** [review]
Everything seems good except annoFusePerSample script. Sine annofuse ought to already have been built, it should only try to install annofuse if for some reason it is being run outside the docker env

File: annoFuse/0.91.0/annoFusePerSample.R [hunk 1]
```diff
@@ -0,0 +1,51 @@
+install.packages("/opt/formatFusionCalls/annoFuse_0.90.0.tar.gz",repos = NULL)
```
[the remark this answers is not in the document]
Comment (sakshamphul) [reply annoFuse/0.91.0/annoFusePerSample.R:1]: Nice catch! I have update the code

**Review by migbro — APPROVED** [review]
Thanks for the fix!

**Pull request merged**